src/brevitas/core/function_wrapper/clamp.py

```diff
@@ -73,3 +79,136 @@ def __init__(self, min_val: float) -> None:
     @brevitas.jit.script_method
     def forward(self, x: Tensor):
         return x.clamp_min(self.min_val)
+
+
+class FloatClamp(brevitas.jit.ScriptModule):
+    """"
+    ScriptModule for clamping minifloat formats to their inf/NaN implementations.
+
+    Currently, inf/NaN codes have to be encoded through the mantissa.
+    I.e. setting inf to 1101.111 (E4M3) is not a valid code.
+    """
+
+    def __init__(
+            self,
+            exponent_bit_width: int,
+            mantissa_bit_width: int,
+            exponent_bias: int,
+            tensor_clamp_impl: Module = TensorClamp(),
+            nan_values: Optional[Tuple[str]] = None,
+            inf_values: Optional[Tuple[str]] = None,
+            saturating: bool = False) -> None:
```

Comment:
```suggestion
            exponent_bit_width: int,
            mantissa_bit_width: int,
            exponent_bias: int,
            tensor_clamp_impl: Module,
            fpx_clamp_impl: Module,
            nan_values: Optional[Tuple[str]] = None,
            inf_values: Optional[Tuple[str]] = None,
            saturating: bool = False) -> None:
```

And then remove lines 123-124

Reply:
Similarly with MaxFloatNaNInt, but that's a bit more challenging. We will do that later once we have more structures for these datatypes. Add a todo: Revisit this implementation with dependency injection